scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/SynchronousSchedulerAgent.java

```diff
@@ -91,8 +92,18 @@ public void schedule(final ScheduleRequest request, final ResultsProtocol.Writer
       ensurePlanRevisionMatch(specification, planMetadata.planRev());
       //create scheduler problem seeded with initial plan
       final var schedulerMissionModel = loadMissionModel(planMetadata);
-      final var planningHorizon = new PlanningHorizon(Time.fromInstant(specification.horizonStartTimestamp().toInstant()), Time.fromInstant(specification.horizonEndTimestamp().toInstant())) ;
-      final var problem = new Problem(schedulerMissionModel.missionModel(), planningHorizon, new SimulationFacade(planningHorizon, schedulerMissionModel.missionModel()), schedulerMissionModel.schedulerModel());
+      final var planningHorizon = new PlanningHorizon(
```

Comment:
I'm not a fan of unrelated formatting changes cluttering the diff of a PR - that said, I do think you have improved the formatting here.

@patkenneally what's our policy on things like this?

Reply:
I was thinking we can set up a standard formating config file for java and javascript and have a PR just for formatting changes.

ex.

I know in javascript you can 'format on save' and open all the javascript files and save them which will format them. Then make a PR with those changes. It will be a simple approve because no logic was changed just the format.

Reply:
To paraphrase Pat's statement from standup: 
> to the extent possible, formatting changes should not be mixed with functional changes in the same commit.

So it's okay to make this change, but it should be in its own commit.

My own thoughts here: changes fall into one of two (ish) categories:
1. Changes that affect the behavior of the program
2. Refactoring / Reformatting

From a reviewer's perspective, reviewing a PR is much more difficult when the two kinds of changes are mixed together. Keeping them separate allows the reviewer to focus on one of these things at a time:
1. Checking that the new behavior of the program is desirable
2. Checking that the changes did not change the behavior of the program

Expecting a reviewer to do a mix of the two at the same time puts a lot of cognitive load on the reviewer, and results in a more difficult, and lower quality review process.

Now, this PR is far from egregious in that sense - the changes are fairly limited and not too hard to review (but since I was around for the writing of a lot of these changes, my opinion is skewed)

Reply:
@goetzrrGit I do like the idea of setting up an autoformatter for Java, but I'd like to get buy-in from the team at large first. I do not have strong opinions about what the correct format is, but I'm sure other people do.

Reply:
@mattdailis As you've said above. 

For this PR and generally for Aerie's repositories; my preference here is that a quick interactive rebase be done separating out the formatting changes from the functional ones. Also, from the initial look through (I'm coming back for a proper review after my next meeting), there are multiple stages in the development of this work which might be better communicated in separate commits. Breaking out the changes into perhaps 3 (maybe 4) stages which walk the reviewer through each change to the final functionality, would greatly help reviewers. 

I want to remind us all of the standard and reasons for the way we build up PRs. The following are linked from our contributing.md. 
[How to write a good commit message](https://chris.beams.io/posts/git-commit/)
[Telling Stories Through Your Commits](https://blog.mocoso.co.uk/talks/2015/01/12/telling-stories-through-your-commits/)
[How to Make Your Code Reviewer Fall in Love with You](https://mtlynch.io/code-review-love/)